frontend/src/app/_components/creator-list/creator-list.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="creator-list row">
+    <div class="col-md-4 col-12 col-sm-6 col-lg-3" *ngFor="let creator of creators">
```

Comment:
I would list these in order to make it more readable. Doesn't matter if it's col 12 col-sm-6 col-md-4 col-lg3 or the opposite order.

Reply:
done